css/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+	background-color: #efefef;
+	margin: 0px;
+}
+h1 {
+	text-align: center;
+	font-family: 'Raleway', sans-serif;
+	font-size: 68pt;
+	margin: 20px;
+	color: #663399;
+}
+nav {
+	text-align: center;
+	background-color: grey;
+	padding: 10px;
+}
+a {
+	text-decoration: none;
+	color: black;
+	margin-right: 25px;
+	margin-left: 25px;
+	font-family: 'Raleway', sans-serif;
```

Comment:
:hammer: It looks like you want the Raleway font for the whole page. If so, then just put it on the `body`.